presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -39,6 +39,8 @@ Property Value                                     Description
                                                    :doc:`/sql/revoke` commands.
 ================================================== ============================================================
 
+Ensure that authentication has been enabled when enabling authorization checks.
```

Comment:
This makes sense. Should we mention what happens if you don't do that? Something like "Without authentication, you'll see XXX error."